core/src/main/java/org/apache/mahout/common/distance/EuclideanDistanceMeasure.java

```diff
@@ -29,26 +31,28 @@
  * squared differences.
  */
 public class EuclideanDistanceMeasure extends SquaredEuclideanDistanceMeasure {
-  
+  Logger log = LoggerFactory.getLogger(EuclideanDistanceMeasure.class);
   @Override
   public double distance(Vector v1, Vector v2) {
     double largeDistanceValue = 1000000;
-    int [] fieldNumbersThatMustMatch = {3}; // Field 3 is dwellingType
+    int [] fieldNumbersThatMustMatch = {6}; // Field 3 is dwellingType
```

Comment:
You are correct Marcel.  This PR can be deleted.  The branch we had been working on was cdh4-0.6_4.0.1_euclid.  I'll see if I can change the project to point to that one by default and I'll remove this PR.